eos-qogir-icons/PKGBUILD

```diff
@@ -15,23 +14,40 @@ optdepends=()
 provides=('qogir-icon-theme')
 conflicts=('qogir-icon-theme' 'qogir-icon-theme-git')
 options=('!strip')
-source=("https://github.com/vinceliuice/Qogir-icon-theme/archive/refs/tags/${_srcver}.tar.gz"
-	"install.patch"
-	"start-here-symbolic.svg"
-	"logout-purple.svg")
-sha256sums=('618768d3592238f0d11379b4a0261434c332bb63c30d3ea85ea26f80bf199043'
-            'c2ac10ce2f0f6bd1260e6580e3dfba3b2eb9b857f20bafabb5f86aa76faa51e0'
+
+source=(
+    "git+https://github.com/vinceliuice/Qogir-icon-theme.git"
+    "install.patch"
+    "start-here-symbolic.svg"
+    "logout-purple.svg"
+)
+
+sha256sums=('SKIP'
```

Comment:
We shouldn't pull the latest without a checksum.

If these changes aren't in a tag, pull off a specific commit.

Reply:
Yeah, pinning it to c633057ba0d27a504b3255144071c9691ed0264a